liquibase-standard/src/main/java/liquibase/integration/spring/SpringLiquibase.java

```diff
@@ -366,6 +368,10 @@ protected Liquibase createLiquibase(Connection c) throws LiquibaseException {
             ((DerbyDatabase) liquibase.getDatabase()).setShutdownEmbeddedDerby(false);
         }
 
+        if (customizer != null) {
+            customizer.customize(liquibase);
+        }
+
 		return liquibase;
```

Comment:
The file already used a mix of tab and space based indentation (if statement is aligned with the closing bracket from the statement above)

Reply:
I’m an external contributor too, lets see what the maintainers have to say :)